automation/jleechanorg_pr_automation/orchestrated_pr_runner.py

```
@@ -0,0 +1,406 @@
+
+
+def dispatch_agent_for_pr(dispatcher: TaskDispatcher, pr: Dict, agent_cli: str = "claude") -> bool:
+    repo_full = pr.get("repo_full")
+    repo = pr.get("repo")
+    pr_number = pr.get("number")
+    branch = pr.get("branch")
+
+    if not all([repo_full, repo, pr_number is not None, branch]):
+        log(f"Skipping PR with missing required fields: {pr}")
+        return False
+    assert branch is not None and pr_number is not None
```

Ruff (0.14.8)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)